device/wormhole/wormhole_implementation.h

```diff
@@ -206,7 +206,7 @@ static constexpr uint32_t TENSIX_SOFT_RESET_ADDR = 0xFFB121B0;
 
 class wormhole_implementation : public architecture_implementation {
    public:
-    tt::ARCH get_architecture() const override { return tt::ARCH::WORMHOLE; }
+    tt::ARCH get_architecture() const override { return tt::ARCH::WORMHOLE_B0; }
```

Comment:
Oh this was probably wrong. I guess it didn't even matter before.
No action here.

Reply:
Maybe it was used, just a lot of checks were so it made the thing above go away

```cpp
if (arch == tt::ARCH::WORMHOLE || arch == tt::ARCH::WORMHOLE_B0) 
```